rust_dev_preview/custom_retries/src/bin/set_retries.rs

```diff
@@ -76,7 +76,7 @@ async fn main() -> Result<(), Error> {
         config::Builder::from(&shared_config)
             // Set max attempts.
             // If tries is 1, there are no retries.
-            .retry_config(RetryConfig::standard().with_max_attempts(tries))
+            .retry_config(RetryConfig::new().with_max_attempts(tries))
```

Comment:
Hmm... This is a problem. `RetryConfig::standard` is actually correct for the latest SDK, but we've been delayed for a couple of weeks on releasing it.

Reply:
Ok, reverted this back and will override the push to accommodate for the breaking change.